release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -102,6 +102,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 [id="ocp-4-7-monitoring"]
 === Monitoring
 
+[id="ocp-4-7-monitoring-api-performance-dashboard"]
+==== New API performance monitoring dashboard
+
+The API performance dashboard is now available from the web console. This dashboard can be used to help troubleshoot performance issues with the Kubernetes API server or the OpenShift API server. You can access the API performance dashboard from the web console by navigating to *Monitoring* -> *Dashboards* and selecting the *API Performance* dashboard.
+
+This dashboard provides API server metrics, such as:
+
+* Request duration
+* Request rate
+* Request latency
+* Request termination
+* Requests in flight
+* Long running requests
```

Comment:
s/Long running requests/Long-running requests
Not sure if this is matching the dashboard GUI? If so, then disregard this suggestion.

Reply:
Nope, not trying to match GUI labels, thanks!